manifest.json

```diff
@@ -139,7 +71,79 @@
         "type":"boolean",
         "default": false,
         "description": "Log all commands, but do not execute."
-    }
+    },
+    "gear-run-bids-validation": {
+        "type": "boolean",
+        "default": false,
+        "description": "Run BIDS validator prior to attempting analysis."
+    },
+    "gear-abort-on-bids-error": {
+        "type": "boolean",
+        "default": false,
```

Comment:
Why default to false?

Reply:
Mainly b/c I wasn't sure where all this would get carried through and I wanted the script to be able to produce errors. Definitely makes more sense to be true after more dev!